_commands/basics/man.md

```diff
@@ -0,0 +1,82 @@
+---
+---
+
+man
+--
+
+`man` allows users to format/display the user "manual" that is built into Linux. This manual holds a catalogue of commands and other aspects of the Linux OS.
+
+~~~ bash
+$ man [command]
+$ man ls
+$ man cd

```

Comment:
`man cd` is a bit of a confusing example since it opens a manpage for all the shell builtins.

I think it would be neat to show as a second example something like `man strcmp` so that it's clear that you can look up both commands and C library functions.